package.json

```diff
@@ -71,8 +73,10 @@
     "jscs": "^2.8.0",
     "jshint": "^2.8.0",
     "mocha": "^2.1.0",
+    "phantomjs": "^1.9.19",
```

Comment:
Yes, but it will only use the phnatomjs in your PATH, when it has the same version: https://github.com/Medium/phantomjs/blob/master/install.js#L466-L484

E.g. when we install phantomjs 2.0 manually on appveyor, this would overwrite it and always use 1.9.